front/app/containers/Admin/projects/project/participation/ParticipationDateRange.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+import { Box, Text } from '@citizenlab/cl2-component-library';
+import moment, { Moment } from 'moment';
+import { useParams } from 'react-router-dom';
+
+import DateRangePicker from 'components/admin/DateRangePicker';
+
+import { useIntl } from 'utils/cl-intl';
+
+import messages from './messages';
+import ParticipationReportPreview from './ParticipationReportPreview';
+
+const ParticipationDatesRange = ({
```

Comment:
I know I'm repeating myself with this component for the two screens but as we will potentially add diverging filters moving forward, I'd rather keep this separate and abstract it a bit later as we know more.